specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/loadBalancer.json

```diff
@@ -2736,6 +2809,51 @@
       },
       "description": "Individual port mappings for inbound NAT rule created for backend pool."
     },
+    "LoadBalancerHealthPerRule": {
+      "type": "object",
+      "properties": {
+        "up": {
```

Comment:
>up [](http://example.com/codeflow?start=9&length=2)

'up' is too generic. please make this property name be intuitive enough to understand reading it.